seed.js

```diff
@@ -1,5 +1,4 @@
-const db = require('./db/model/puppies');
-const Puppy = db.model.puppies;
+const { db, Puppy } = require('./db/model/puppies');
```

Comment:
Same issue as in the router – your `puppies` module exports an object with `db` and `Puppy` props, so we need to use destructuring to "extract" the `db` (and/or `Puppy`) part(s).